app/networkapi/features/apps.py

```diff
@@ -2,4 +2,8 @@
 
 
 class FeaturesConfig(AppConfig):
-    name = 'features'
+    name = 'networkapi.features'
+    verbose_name = 'features'
+
+    def ready(self):
+        from networkapi.features import signals  # noqa
```

Comment:
not having side effects is probably a good idea. I only just now realised it is literally my main point of hatred around django, too.